README.md

```diff
@@ -4,15 +4,70 @@
 
 [![Chi-chan bot Container](https://github.com/seker212/Discord_bot/actions/workflows/Chi-chan_Container.yml/badge.svg?branch=master&event=push)](https://github.com/seker212/Discord_bot/actions/workflows/Chi-chan_Container.yml)
 
-## Running locally
+## Table of contents
 
-Clone this repo with checkout to branch `master-2.0`. Run the `DiscordBot` project. :bangbang: Before running remember to add to the main direcotry file `token.txt` with discord bot token.
+1. [Running bot](#tc1)
+    1. [Important informations](#tc11)
+    2. [Local run](#tc12)
+    3. [Doker run](#tc13)
+        1. [Enviroment informations](#tc131)
+        2. [Using docker cli](#tc132)
+        3. [Using docker compose](#tc133)
+2. [Debug and development](#tc2)
+    1. [Database schemas](#tc21)
 
-## Running in docker
+## Running bot <a id="tc1"></a>
 
-### Using docker compose
+This part is about how to run bot depending on enviroment used.
 
-Container can be run using provided docker compose file provided example provided in `docker-compose-template.yaml`, or from the example below.
+### Imporatnt informations <a id="tc11"></a>
+
+Before trying to run you need to aquire token from [Discord](https://discord.com/developers/applications). 
+
+### Local run <a id="tc12"></a>
+
+Running locally is not advised as requires use of Visual Studio and installation of .Net 6. For debbuging it is prefered method.
+
+1. Clone this repo with checkout to branch `master-2.0`.
+2. Open `DiscordBot.sln` with Visual studio
+3. Place `token.txt` file in main folder with token aqured earlier.
+4. Run the `DiscordBot` project
+
+If there are some issues with packages try reimporting nu-get.
```

Comment:
nu-get? XD

Reply:
Forgor how to write this and it was late